key-chord.el

```diff
@@ -337,15 +353,17 @@ Please ignore that."
               (when (bound-and-true-p eldoc-mode)
                 (eldoc-pre-command-refresh-echo-area))
 
-	      (sit-for delay 0 'no-redisplay))
+	      (sit-for delay 'no-redisplay))
```

Comment:
This seems to fix a bug and should be done in a separate commit.

Reply:
OK, I make a separate commit.